styles/file-patch-list-view.less

```diff
@@ -30,6 +30,7 @@
 
   &-path {
     flex: 1;
+    word-break: break-all;
```

Comment:
Agreed, I think that looks better. @will-stone if you'd like to make that change, I'll get this merge — or I can make it, either way. Thanks! ✨